client/assets/stylesheets/jetpack-cloud.scss

```diff
@@ -31,11 +31,19 @@
 	}
 
 	.layout__primary {
```

Comment:
Instead of using `.layout__primary' can we have a different wrapper for the whole scan layout to archive this? 
This file will eventually go away :( when we just have a single entry point.